f5_openstack_agent/tests/functional/testlib/bigip_client.py

```diff
@@ -0,0 +1,90 @@
+# coding=utf-8
+# Copyright 2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+import re
+import urllib
+
+from f5.bigip import ManagementRoot
+from f5_openstack_agent.lbaasv2.drivers.bigip import cluster_manager
+from f5_openstack_agent.lbaasv2.drivers.bigip import resource_helper
+from f5_openstack_agent.lbaasv2.drivers.bigip import system_helper
+
```

Comment:
Another import inconsistency.

Reply:
With respect to imports, e.g. in bigip_client, those imports should migrate to the relative import pattern.   I'll write up a complete policy for import syntax.  Since we're using "hacking" from "openstack", we won't be able to deviate too far from the community.